doc/dualtor/active_active_hld.md

```diff
@@ -486,7 +488,65 @@ Previously, at a high level, when the mux port comes to standby, the MuxOrch add
 A change to skip the installation of ingress drop ACL rule when toggling standby is introduced to forward the upstream traffic with best effort. This is because that, though the mux port is already in standby state in this period, the removal of the ingress drop ACL could allow the upstream traffic to reach the ToR and to be possibly forwarded by the ToR.
 
 ### 3.9 Command Line
-TBD 
+This part only covers the command lines and options for active-active dualtor.
+
+#### 3.9.1 Show mux status
+`show mux status` returns the mux status for mux ports:
+  * `PORT`: mux port name
+  * `STATUS`: current mux status, could be either `active` or `standby`
+  * `SERVER_STATUS`: the mux status read from mux server as the result of last toggle
+    * `active`: mux server returned `active` as the result of last toggle to `active`
+    * `standby`: mux server returned `standby` as the result of last toggle to `standby`
+    * `unknown`: last toggle failed to switch the mux server status, or failed to read the status from the mux server
+    * `error`: last toggle failed to switch the orchagent status
+  * `HEALTH`: mux port health
+    *  `healthy`: it means that the ToR could receive link probe replies from the mux server, the following conditions must be satisfied for a mux port to be `healthy`:
+       * port status is `up`
+       * could receive replies for self link probes
+       * current mux status(`STATUS`) should match server status(`SERVER_STATUS`) or server status is `unknown`
+       * default route to T1s is present
+    *  `unheathy`: any of the above `healthy` conditions is broken
+  * `HWSTATUS`: check if current mux status matches server status
+    * `consistent`: `STATUS` matches `SERVER_STATUS`
+    * `inconsistent`: `STATUS` doesn't matches `SERVER_STATUS`
+    * `absent`: `SERVER_STATUS` is not present
+  * `LAST_SWITCHOVER_TIME`: last switchover timestamp
+
+```
+$ show mux status
+PORT        STATUS    SERVER_STATUS    HEALTH    HWSTATUS    LAST_SWITCHOVER_TIME
+----------  --------  ---------------  --------  ----------  ---------------------------
+Ethernet4   active    active           healthy   consistent  2023-Mar-27 07:57:43.314674
+Ethernet8   active    active           healthy   consistent  2023-Mar-27 07:59:33.227819
+```
+
+#### 3.9.2 Show mux config
+`show mux config` returns the mux configurations:
+  * `SWITCH_NAME`: peer switch hostname
+  * `PEER_TOR`: peer switch loopback address
+  * `PORT`: mux port name
+  * `state`: mux mode configuration
+    * `auto`: enable failover logics for both self and peer
+    * `manual`: disable failover logics for both self and peer
+    * `active`: if current mux status is not `active`, toggle the mux to `active` once, then work in `manual` mode
```

Comment:
"then work in manual mode"

-- is this accurate? If periodical gRPC querying find server state == standby, wouldn't we toggle again to active?

Reply:
> "then work in manual mode"
> 
> -- is this accurate? If periodical gRPC querying find server state == standby, wouldn't we toggle again to active?

Yes, thr ToR will toggle to active in your case, this is the reaction to the mux hardware. But after config mux `active` or `standby`, the ToR will not react to any of the failover scenarios.